tests/scripts/create_coverage_report.py

```diff
@@ -35,6 +35,11 @@ def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
         lcov_list.append("-a")
         lcov_list.append(f"{merge_dir}/{file_path.name}")
 
+    # Skip creating coverage report when no .info files are found
+    if len(lcov_list) == 1:
```

Comment:
Hmm, how is it possible to get into this error? Aren't there always info files, when testing with code coverage is enabled? And isn't line https://github.com/eclipse-bluechi/bluechi/blob/main/tests/scripts/create_coverage_report.py#L61 preventing this error when code coverage is disabled?

Reply:
As we discussed, I updated the commit message to describe how it can happen that no .info files are existing due to provisioning, preparation or execution issues. The new tmt version breaking local provisioning Ubuntu is one example (https://github.com/eclipse-bluechi/bluechi/issues/734).